index.html

```diff
@@ -94,17 +117,46 @@ <h3><a class="textdecoration font1" href="#">Tonic</a></h3>
 
             </div>
 
+            <div class="article1div show2">
+
+                <h3><a class="textdecoration font1" href="#">Tonic</a></h3>
+
+                <ul>
+
+                    <li><a class="textdecoration font2" href="#">CANOPY</a></li>
+                    <li><a class="textdecoration" href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
+                    <li><a class="textdecoration font3" href="#">Back End Dev</a></li>
+                    <li class="textdecoration"><a href="#"><img src="./img/mid_dot.jpg" alt=""></a></li>
+                    <li><a class="textdecoration font3" href="#">2015</a></li>
+
+                </ul>
+
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+                <ul class="gridbox1">
+
+                    <li class="background1"><a class="textdecoration font4" href="#">html</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">Ruby on rails</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">css</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">javaScript</a></li>
+
+                </ul>
+
+                <button type="submit"><a class="font5 textdecoration" href="#">See Project</a></button>
```

Comment:
https://github.com/EmekaEmmanuel/portfolio_mobile_day3/blob/d910d203c2003cf8d6a80ff48513ee9ef14b030e/index.css#L230-L239

Please notice that the Figma's button doesn't contain a border 😉:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/199866624-30bc62b1-9728-4813-9d20-a7e0d0940b35.png)  | ![image](https://user-images.githubusercontent.com/42799579/199866380-430140a4-8b35-430b-9995-5b085196dbb6.png)  |

To remove it, you could do:
```css
.button {
    border: 0;
}
```